src/tempo.py

```diff
@@ -132,21 +147,21 @@ def get_receiver_url(self, protocol: ReceiverProtocol, ingress: TraefikRouteRequ
         return f"{url}:{receiver_port}"
 
     def _build_server_config(self):
-        server_config = {
-            "http_listen_port": self.tempo_http_server_port,
+        server_config = Server(
```

Comment:
all other types have a Config suffix, I think this should be ServerConfig as well.